apps/storybook/src/HeatmapVisDisplay.stories.tsx

```diff
@@ -98,8 +98,65 @@ WithAxisLabels.args = {
 
 export const WithAnnotation: Story<HeatmapVisProps> = (args) => (
   <HeatmapVis {...args}>
-    <Annotation x={30} y={18} style={{ width: '200px', color: 'white' }}>
-      An annotation
+    <Annotation x={10} y={15} style={{ color: 'white' }}>
+      HTML annotation positioned at (10, 15)
+    </Annotation>
+    <Annotation
+      x={10}
+      y={5}
+      center
+      style={{
+        width: 180,
+        color: 'white',
+        textAlign: 'center',
+      }}
+    >
+      Another annotation, <strong>centred</strong> on (10, 5)
+    </Annotation>
+    <Annotation
+      x={25}
+      y={10}
+      style={{
+        display: 'flex',
+        alignItems: 'center',
+        width: 320,
+        height: 50,
+        color: 'white',
+        fontSize: '0.875rem',
+        textAlign: 'center',
+      }}
+    >
+      <>
+        <p
+          style={{
+            flex: '1 1 0%',
+            margin: 0,
+            padding: '0.5rem',
+            border: '10px solid pink',
```

Comment:
:nauseated_face:

Reply:
You're right, `pink` and `white` did not work together. I went with `pink` and `papayawhip` instead.

![image](https://user-images.githubusercontent.com/2936402/165058596-1a16c426-9209-4573-a9a9-bab2e40824e3.png)